Loadify/View/Components/CustomTextField.swift

```diff
@@ -13,10 +13,12 @@ struct CustomTextField: View {
     @Binding var text: String
     
     var placeHolder: String
+    var onPaste: () -> Void // Add a closure to handle paste action
```

Comment:
We're making things more complicated by using closures here. This can be simplified. First, we don't need any closures in this context. You can go ahead and remove them.

Reply:
Agreed!